Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -46,42 +56,49 @@ class InAppMessageViewTest: UnitTest {
 
     // MARK: Display in-app message
 
-    func test_displayInAppMessage_givenNoMessageAvailable_expectDoNotDisplayAMessage() {
+    @MainActor
+    func test_displayInAppMessage_givenNoMessageAvailable_expectDoNotDisplayAMessage() async {
         queueMock.getInlineMessagesReturnValue = []
 
         let inlineView = InAppMessageView(elementId: .random)
 
-        XCTAssertNil(getInAppMessageWebView(fromInlineView: inlineView))
+        XCTAssertFalse(isDisplayingInAppMessage(inlineView))
```

Comment:
I changed some of the `getInAppMessageWebView` calls to the new `isDisplayingInAppMessage` to make the test function more accurate checking if the inline View is displaying a in-app message. By testing the height of the View is not 0, we are more certain that the inline View is visible/displayed to the user.